integration_test/cases/browser/click_button_test.exs

```diff
@@ -1,6 +1,8 @@
 defmodule Wallaby.Integration.Browser.Actions.ClickButtonTest do
   use Wallaby.Integration.SessionCase, async: true
 
+  alias Wallaby.Integration.PageObjects.IndexPage
```

Comment:
Small note but it might be misleading to call these `Objects`. I just call them `Pages` typically.